### PR TITLE
fix TLS docker-dind problem from version 18.09+

### PR DESCRIPTION
**Pull request by vlauciani**
This PR resolve the problem:
- https://github.com/laradock/laradock/issues/2223

I'm working on new PR to use the last version of `dind`: 18.09+

##### I completed the 3 steps below:

- [X] I've read the [Contribution Guide](http://laradock.io/contributing).
- [ ] I've updated the **documentation**. (refer to [this](http://laradock.io/contributing/#update-the-documentation-site) for how to do so).
- [X] I enjoyed my time contributing and making developer's life easier :)

### CODE REVIEW

**Comment by vlauciani** [comment]
With commit:
- https://github.com/laradock/laradock/pull/2224/commits/e3c89b861bc194a69f41190934879bd5dad8e2f1

I've implemented the TLS certificates; it is now a default in `18.09+`.

**Comment by vlauciani** [comment]
Please, merge! It solve a TLS docker-dind problem!

thank you

File: docker-compose.yml [hunk 1]
```diff
@@ -1168,9 +1178,11 @@ services:
 ### Docker-in-Docker ################################################
     docker-in-docker:
       image: docker:dind
+      hostname: docker-in-docker
```
Comment (edentsai) [inline docker-compose.yml]: docker:dind image will automatically  generate TLS certificates with `DOCKER_TLS_SAN` env variable,  
for example:

```diff
  services:
    docker-in-docker:
-     hostname: docker-in-docker
+     environment:
+       DOCKER_TLS_SAN: DNS:docker-in-docker
```

Reference: https://github.com/docker-library/docker/blob/92d278e671f3/18.09-rc/dind/dockerd-entrypoint.sh#L8-L20

Comment (vlauciani) [reply docker-compose.yml]: Ok, thank you.
I've update the PR.

**Comment by vlauciani** [comment]
Someone knows the delay with this PR? It fix a security problem!

**Comment by vlauciani** [comment]
I'm working on a new PR; I'll submit it again in a few days.

**Pull request closed without merging**